holidayextras/brands
# OPS-2348 Airparks brand on tripapp

#### What does this PR do?
makes the airparks brand work with tripapp.

#### How can this be tested?

- have both aps-brands branch pulled down on brands and aps-brandingv2 on tripapp.

- in the tripapp terminal run : npm install @holidayextras/brand-airparks.

- you might get an unmet peer dependency error when installing, if you do update your grunt by doing npm install grunt@0.4

- then from tripapp terminal type npm link ../brands/brands/airparks (depending on your directory to the brands repo).

- make sure your host files are set up to have v2.holidayextras.co.uk and v2.airparks.co.uk. 

- if your url automatically goes to https://... go to chrom://net-internals and delete the url to stop it auto redirecting you to https.

#### What does this PR do? (please provide any background)

#### Any tech debt?

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---
